partiql-tests-kotlin-test-runner/src/test/kotlin/org/partiql/tests/kotlintestrunner/Schema.kt

```diff
@@ -0,0 +1,43 @@
+package org.partiql.tests.kotlintestrunner
+
+import com.amazon.ion.IonStruct
+import com.amazon.ion.IonValue
+import org.partiql.lang.eval.CompileOptions
+
+data class Namespace(
+    var env: IonStruct,
+    val namespaces: MutableList<Namespace>,
+    val testCases: MutableList<TestCase>,
+    val equivClasses: MutableMap<String, List<String>>
+)
+
+data class EquivalenceClass(val id: String, val statements: List<String>)
+
+sealed class Assertion {
+    data class EvaluationSuccess(val expectedResult: IonValue) : Assertion()
+    object EvaluationFailure : Assertion()
+    // TODO: other assertion and test categories: https://github.com/partiql/partiql-tests/issues/35
```

Comment:
Since the test runner came out of discussion related to evaluation tests, I decided to only test the evaluation and evaluation equivalence tests in this PR. Other test categories will be added in a future PR as part of https://github.com/partiql/partiql-tests/issues/35